ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -128,7 +131,9 @@ case class FieldStats(field: String,
 
 /** Big diff between two data sets given a primary key. */
 class BigDiffy[T](lhs: SCollection[T], rhs: SCollection[T],
-                  diffy: Diffy[T], keyFn: T => MultiKey) {
+                  diffy: Diffy[T], keyFn: T => MultiKey)
+                 (implicit coder0: Coder[(String, T)],
+                  coder1: Coder[(MultiKey, (String, T))]) {
```

Comment:
It should be fine as long as we have a coder for `T` and `MultiKey` right? I think `MultiKey` it should also automatically derive since it's a case class so I would try with just a `T: Coder` context bound